### PR TITLE
Skip build for python 3.7

### PR DESCRIPTION
**Pull request by astamminger**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by astamminger** [comment]
@ltalirz Circus still not there (neither on Linux nor on OSX) and "nothing to test" for aiida-core package. I'll keep this PR open and have a look if you don't mind.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -59,23 +59,24 @@ requirements:
...
+    - aiida.common
+    - aiida.manage
+    - aiida.restapi
+    - aiida.schedulers
+    - aiida.transports
+  commands:
+    - reentry scan -r aiida
+    - verdi --help
+    - verdi --version
+#    - verdi status
```
Comment (ltalirz) [inline recipe/meta.yaml]: was this commented out because it returns exit code 1?
If yes, then just remove this commented line

**Comment by ltalirz** [comment]
@conda-forge-admin , please rerender

**Comment by ltalirz** [comment]
Fine to merge after re-rendering & deciding what to do with the `verdi status` line

**Comment by ltalirz** [comment]
@astamminger it's now failing at the import tests, which makes sense

File: recipe/meta.yaml (hunk 1 above)
Comment (astamminger) [reply recipe/meta.yaml]: Yes, I just wanted to know if the tests pass and ask you afterwards if you know about that verdi status error. I also do not know if it is best practice to put that reentry command in there...

**Comment by astamminger** [comment]
> @astamminger it's now failing at the import tests, which makes sense

What do you mean?

**Comment by ltalirz** [comment]
> What do you mean? (They should pass)

Sorry, somehow I was looking at an old commit.
It's strange - in my web view of the PR, the commit before the rerendering one is "Perform reentry scan before test execution" which has this failure due to `import aiida.scheduler`

File: recipe/meta.yaml (hunk 1 above)
Comment (ltalirz) [reply recipe/meta.yaml]: > I also do not know if it is best practice to put that reentry command in there...

I think it's fine.

**Comment by astamminger** [comment]
> > What do you mean? (They should pass)
> 
> Sorry, somehow I was looking at an old commit.
> It's strange - in my web view of the PR, the commit before the rerendering one is "Perform reentry scan before test execution" which has this failure due to `import aiida.scheduler`

I see, i just noticed that as well. It's a strange ordering...

**Comment by astamminger** [comment]
FYI: The verdi status command fails when checking the profile: 
```+ verdi status
Traceback (most recent call last):
 ✗ profile:     Unable to read AiiDA profile
  File "/home/andreas/anaconda3/4.2.0/envs/python3.6/conda-bld/aiida-core_1552943018171/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/bin/verdi", line 11, in <module>
    sys.exit(verdi())
  File "/home/andreas/anaconda3/4.2.0/envs/python3.6/conda-bld/aiida-core_1552943018171/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/andreas/anaconda3/4.2.0/envs/python3.6/conda-bld/aiida-core_1552943018171/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/andreas/anaconda3/4.2.0/envs/python3.6/conda-bld/aiida-core_1552943018171/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/andreas/anaconda3/4.2.0/envs/python3.6/conda-bld/aiida-core_1552943018171/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/andreas/anaconda3/4.2.0/envs/python3.6/conda-bld/aiida-core_1552943018171/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/andreas/anaconda3/4.2.0/envs/python3.6/conda-bld/aiida-core_1552943018171/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.6/site-packages/aiida/cmdline/commands/cmd_status.py", line 64, in verdi_status
    raise exc  # without a profile, we cannot access anything
  File "/home/andreas/anaconda3/4.2.0/envs/python3.6/conda-bld/aiida-core_1552943018171/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.6/site-packages/aiida/cmdline/commands/cmd_status.py", line 59, in verdi_status
    profile = manager.get_profile()
  File "/home/andreas/anaconda3/4.2.0/envs/python3.6/conda-bld/aiida-core_1552943018171/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.6/site-packages/aiida/manage/manager.py", line 91, in get_profile
    config = get_config()
  File "/home/andreas/anaconda3/4.2.0/envs/python3.6/conda-bld/aiida-core_1552943018171/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.6/site-packages/aiida/manage/configuration/__init__.py", line 41, in get_config
    CONFIG = load_config()
  File "/home/andreas/anaconda3/4.2.0/envs/python3.6/conda-bld/aiida-core_1552943018171/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.6/site-packages/aiida/manage/configuration/utils.py", line 45, in load_config
    raise exceptions.MissingConfigurationError('configuration file {} does not exist'.format(filepath))
aiida.common.exceptions.MissingConfigurationError: configuration file /home/andreas/.aiida/config.json does not exist
```
Any chance that this is already known?

**Comment by ltalirz** [comment]
Well... I thought re-raising the exception made sense since 

```
 File "/home/andreas/anaconda3/4.2.0/envs/python3.6/conda-bld/aiida-core_1552943018171/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.6/site-packages/aiida/cmdline/commands/cmd_status.py", line 64, in verdi_status
    raise exc  # without a profile, we cannot access anything
```

however, this use case shows that this is not actually what we want.
It's a very easy fix: instead of this
https://github.com/aiidateam/aiida_core/blob/develop/aiida/cmdline/commands/cmd_status.py#L64

you simply directly `return 0`
Do you want to make the PR?

**Comment by astamminger** [comment]
> Do you want to make the PR?

No no, got ahead, your fix your credit :D

**Comment by astamminger** [comment]
@ltalirz I removed the status test for now.

**Comment by ltalirz** [comment]
the behavior of `verdi status` has been modified in https://github.com/aiidateam/aiida_core/pull/2634

it will still return a non-zero exit status, though, if some required services are not running.
i.e. it should not be added to the tests in the feedstock

**Pull request merged**